ui/apps/moderation/src/components/app-routes.tsx

```diff
@@ -48,7 +50,8 @@ import routes, {
 const AppRoutes: React.FC<RootComponentProps> = props => {
   const loginQuery = useGetLogin();
```

Comment:
we no longer use this hook, replace it with useGetMyProfileQuery from @akashaorg/ui-awf-hooks/lib/generated/hooks-new